config/permissions.yaml

```diff
@@ -13,3 +13,7 @@
     - 100
 "ajax.holidaycredit":
     - 100
+"notifications.index":
```

Comment:
Good thing to provide notifications.index. So you can remove the entry in access with page = notification

Reply:
Thank you. db_data.php and atomicupdate are now updated.